airbyte-integrations/connectors/source-sentry/.coveragerc

```diff
@@ -1,3 +1,3 @@
 [run]
-omit = 
+omit =
     source_sentry/run.py
```

Comment:
Some folks had to also add `main.py` and `__init__.py`, but if the coverage is hit, I guess we're okay /shrug

Reply:
Note to self: this file should be in the template for new connectors